src/condor/backend/operators.py

```diff
@@ -20,16 +20,18 @@
 # constants
 pi = backend_mod.operators.pi
 inf = backend_mod.operators.inf
+nan = backend_mod.operators.nan
 
 # calculus & symbolic
 jacobian = backend_mod.operators.jacobian
-recurse_if_else = backend_mod.operators.recurse_if_else
+if_else = backend_mod.operators.if_else
```

Comment:
Note for release notes, this is a breaking change. Is it worth having a deprecation schedule?

Reply:
Can punt this to an issue for pre-release

Reply:
I think since we're version < 1.0, we don't need to bother with a deprecation schedule, but open to it